src/components/main/note/EditableText.tsx

```diff
@@ -55,15 +67,6 @@ const EditableText = forwardRef(
   },
 );
 
-EditableText.propTypes = {
-  isEditing: PropTypes.bool.isRequired,
-  onToggleEdit: PropTypes.func.isRequired,
-  onChange: PropTypes.func.isRequired,
-  text: PropTypes.string,
-};
-
-EditableText.defaultProps = {
-  text: 'Edit me...',
-};
+EditableText.displayName = 'EditableText';
```

Comment:
Still need this?

Reply:
It seems so. Otherwise the linter complains:
> Component definition is missing display name eslint([react/display-name](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/display-name.md))